src/stcal/ramp_fitting/ols_fit.py

```diff
@@ -22,7 +22,7 @@
 
 
 def ols_ramp_fit_multi(
-        input_model, buffsize, save_opt, readnoise_2d, gain_2d, weighting, max_cores):
+        ramp_class, buffsize, save_opt, readnoise_2d, gain_2d, weighting, max_cores):
```

Comment:
I find it a bit confusing to have the name of an instance of a model be 'ramp_class'. Instead, perhaps something containing the words 'model' and 'ramp' but not mission-specific, and something clearly distinct from anything we already use.

Reply:
The `ramp_class` parameter is an instance of the `RampFitInternal` class, not a data model.  The `RampFitInternal` class inherits no other class and has only `ndarray` and basic data types.  No code beyond `ramp_fit` in `ramp_fit.py` depends on a data model instance.  The class is used in preparation for implementing multiprocessing.  The data models cannot be used with the `multiprocessing` module because they cannot be pickled.  A simpler data structure is required for this.

Reply:
Ok, I understand now. This class could also be useful for other calibration steps.